server/aws-lsp-codewhisperer/src/language-server/chat/telemetry/chatTelemetryController.ts

```diff
@@ -345,7 +357,15 @@ export class ChatTelemetryController {
                                 : ChatInteractionType.Downvote,
                         codewhispererCustomizationArn: this.getCustomizationId(params.tabId, params.messageId),
                     }
-
+                    this.#telemetryService.emitChatInteractWithMessage(
+                        {
+                            ...voteData,
+                            credentialStartUrl: this.#credentialsProvider.getConnectionMetadata()?.sso?.startUrl,
```

Comment:
nit: since TelemetryService has access to `CredentialsProvider`, we can inject `credentialsStartUrl` there